### PR TITLE
feat: 🎸 install wizard for lightning phase

### PR DESCRIPTION
**Pull request by heldrida**

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -8,92 +8,510 @@
...
+    z"
+  /
 
 ART
 
 echo
-echo "👽 Welcome ultra-terrestrial being 👽"
-echo
-echo "We are ready to launch the communication channel with the Fleek Network 📡, but you need to make a choice!"
-echo "There are two options, where option number 1 is to run a Network Node un a Docker Container, or option number 2, installed natively in your operating system."
```
Comment (nicolaspoggi) [inline install:60]: ...is to run a Network Node **in a** Docker Container

File: install [hunk 2]
```diff
@@ -8,92 +8,510 @@
...
-echo "Most Node Operators are using the Docker Stack (if you stick with the recommendations, this is the quickest way to setup 🚀), while other's have a preference for running Natively (might take longer to set up, as it requires building the Ursa CLI binary from the Rust 🦀 source-code)."
```
Comment (nicolaspoggi) [inline install:61]: set up instead of setup

Comment (nicolaspoggi) [reply install:61]: and "others", not "other's"

**Review by nicolaspoggi — COMMENTED** [review]
Some minor text tweaks :D love the ASCII

File: install (hunk 1 above)
Comment (heldrida) [reply install:60]: That's true, but the whole text was removed, thus the `-`.

**Pull request merged**